### PR TITLE
SEARCH-1790: Terratest infra test

### PR DESCRIPTION
**Pull request by uppinder**
For testing:
* Whether EKS cluster was correctly created by Terraform scripts
* Whether the required k8s resources are created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +0,0 @@
-# Elasticsearch deployment on EKS K8s
```
Comment (lplazas) [inline README.md]: Why was this file deleted?

Comment (uppinder) [reply README.md]: MB, deleted the wrong file. Will revert in new commit.
